torchft/manager_test.py

```diff
@@ -483,3 +487,25 @@ def test_manager_numerics(self, client_mock: MagicMock) -> None:
         fut = manager.allreduce(torch.tensor([1.0]))
         result = fut.value()
         torch.testing.assert_close(result, torch.tensor([0.0]))
+
+    @patch("torchft.manager.ManagerClient", autospec=True)
+    def test_quorum_happy_timeouts(self, client_mock: MagicMock) -> None:
+        manager = self._create_manager(use_async_quorum=False)
+        client_mock().should_commit = mock_should_commit
+
+        client_mock().quorum.return_value = (
+            123,  # quorum_id
+            1,  # replica_rank
+            2,  # replica_world
+            "manager address",
+            f"localhost:{self.store.port}",
+            1,  # max_step
+            1,  # max_rank
+            2,  # max_world_size
+            False,  # heal
+        )
+
+        manager.start_quorum(timeout=timedelta(seconds=12))
+        self.assertTrue(manager.should_commit(timeout=timedelta(seconds=23)))
+
+        self.assertEqual(client_mock().quorum.call_count, 1)
```

Comment:
Im not sure if this actually tests that the timeout is passed correctly. Seems to just test that the _clients methods are called at all. Maybe check that quorum was called with the expected timeout?

Reply:
added a check on the kwargs